service-workers/service-worker/fetch-event.https.html

```diff
@@ -17,6 +17,8 @@
     const registration =
         await service_worker_unregister_and_register(t, worker, scope);
     await wait_for_state(t, registration.installing, 'activated');
+
+    // This will happen after all other tests
```

Comment:
Do you want to keep this comment, or revert that too?

Reply:
I'd like to keep that if that's ok. It confused me when reading those tests.

Reply:
That's fine, just making sure it was intentional.